src/components/ContentPreview/ContentPreview.js

```diff
@@ -228,16 +229,19 @@ class ContentPreview extends PureComponent<Props, State> {
      * @return {boolean}
      */
     shouldLoadPreview(prevProps: Props, prevState: State): boolean {
-        const { file }: State = this.state;
+        const { file, fileError }: State = this.state;
         const { file: prevFile }: State = prevState;
         let loadPreview = false;
 
         // Load preview if file version ID has changed
         if (file && file.file_version && prevFile && prevFile.file_version) {
             loadPreview = file.file_version.id !== prevFile.file_version.id;
-        } else {
+        } else if (file) {
```

Comment:
Minor: It seems like this should be an outer check or a guard clause instead, since it's also checked above.

Reply:
nice, I refactored to check for existence of the file first so we only have to check once.